### DISCUSSION TITLE
Undefined variable $exportActive

### DISCUSSION THREAD

**Discussion by syukriyansyah-ipb** [Power-Components/livewire-powergrid#464] [BUG]
I created a new powergrid table with version 3, but when I first call it in view, the error message appears. How to fix this problem?

**Comment by mborowik** [comment]
did you solve the problem?

**Comment by luanfreitasdev** [comment]
Hi!

I believe you have published view files.

Please try to remove the files in `resources/views/vendor/livewire-powergrid` as it is not supported

**Comment by mborowik** [comment]
Hi!
Thank you, problem solved.
